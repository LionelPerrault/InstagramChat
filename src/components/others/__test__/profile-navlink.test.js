import React from 'react'
import ProfileNavLink from '../profile-navlink'
import { create } from 'react-test-renderer'

describe('Profile Navlink', () => {
  it('should match snapshot', () => {
    const tree = create(
      <ProfileNavLink url="/profile/takkar/bookmarks" label="Bookmarks" />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
