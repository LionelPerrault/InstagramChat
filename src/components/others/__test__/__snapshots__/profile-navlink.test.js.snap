// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Navlink should match snapshot 1`] = `
<li>
  <NavLink
    activeClassName="pro_nav_active"
    className="inst_nav"
    exact={true}
    to="/profile/takkar/bookmarks"
  >
    Bookmarks
  </NavLink>
</li>
`;
